.lintstagedrc

```diff
@@ -0,0 +1,5 @@
+---
+"{src,examples,__tests__}/**/*.{re,rei}":
+  - "npm test -- *.js"
+  - "npm run format -- --in-place"
+  - "git add"
```

Comment:
I'm unfamiliar with lintstaged, but generally order matters here. If formatting breaks something it would be staged without testing it with the changes. Also, isn't "npm test" enough? Shouldn't all the tests be run? I'm not clear on why we need arguments to npm test.